newbank/client/ClientDisplay.java

```diff
@@ -0,0 +1,31 @@
+package newbank.client;
+
+import java.util.ArrayList;
+import java.util.List;
+import static java.lang.System.out;
+
+public class ClientDisplay {
+
+    public ClientDisplay(){}
+
+    public static void clientServiceIntroduction(){
+        System.out.println("|-----------------------------------|");
+        System.out.println("|Welcome to NewBank banking service.|");
+        System.out.println("|-----------------------------------|");
+    }
+
+    public static List<String> clientServiceMenu(){
+        List<String> instructionsList = new ArrayList<>();
+        instructionsList.add("|----------------------------|");
+        instructionsList.add(" Please SELECT the following:");
```

Comment:
Maybe I would change this for "Please type the command you wish to execute" to make clear that they need to type the instruction instead of writing the number

Reply:
changes have been applied